### ISSUE TITLE
Last Edited highlight disables On/Off highlight

### ISSUE THREAD

**Issue by muranyia** [ahlstromcj/sequencer64#18]
When I open a pattern for editing, it gets highlighted in blue, which is a cool feature.
Then, when I close the edit window, the highlight stays and if switch the pattern off or on, it stays blue, effectively disabling the black or white background which would appear normally.

My suggestion is to keep the on/off highlight enabled at all times, oin which case if the track is on, it would have a black background with blue color text and preview.

In other words: Off/On =dark-on-light/light-on-dark, Last Edited: white becomes blue.

**Comment by ahlstromcj** [comment]
Yeah, that's kind of crufty and buggy.  I will make this one a priority.  Thanks!

**Comment by ahlstromcj** [comment]
I fixed this sometime in May, I think.  I need to remember to add the issue number to the commit message, sigh.  Anyway, it is fixed and works pretty well I think.

**Comment by muranyia** [comment]
Yes it work well, Thank You!
